plugins/services/src/js/components/modals/ServiceRootGroupModal/utils.tsx

```diff
@@ -28,6 +30,26 @@ export function emptyGroupFormData(): GroupFormData {
   };
 }
 
+export function groupFormDataFromGraphql(data: ServiceGroup): GroupFormData {
+  const quota = data.quota;
+  const enforceRole = findNestedPropertyInObject(quota, "enforced");
+  const cpus = findNestedPropertyInObject(quota, "cpus.limit");
+  const mem = findNestedPropertyInObject(quota, "memory.limit");
+  const disk = findNestedPropertyInObject(quota, "disk.limit");
+  const gpus = findNestedPropertyInObject(quota, "gpus.limit");
+  return {
+    id: data.id || "",
+    enforceRole: enforceRole !== undefined ? enforceRole : true,
+    quota: {
+      force: false,
+      cpus: cpus !== undefined ? cpus : "",
```

Comment:
should we `.toString()` these values since they come out of the GQL as floats but we treat them as strings in the form?